cdk/package.json

```diff
@@ -19,19 +20,20 @@
   },
   "devDependencies": {
     "@guardian/cdk": "v59.3.1",
-    "@guardian/eslint-config-typescript": "1.0.11",
+    "@guardian/eslint-config-typescript": "10.0.1",
     "@guardian/prettier": "2.1.5",
-    "@types/jest": "^29.5.12",
-    "@types/node": "22.5.3",
+    "@types/jest": "29.5.12",
+    "@types/node": "20.11.0",
     "aws-cdk": "2.155.0",
     "aws-cdk-lib": "2.155.0",
     "constructs": "10.3.0",
-    "eslint": "^8.57.0",
-    "jest": "^29.7.0",
-    "prettier": "^2.8.8",
-    "source-map-support": "^0.5.20",
-    "ts-jest": "^29.1.5",
-    "ts-node": "^10.9.2",
-    "typescript": "~4.9.5"
+    "eslint": "8.57.0",
+    "jest": "29.7.0",
+    "prettier": "2.8.8",
+    "source-map-support": "0.5.20",
+    "ts-jest": "29.1.5",
+    "ts-node": "10.9.2",
+    "tslib": "2.6.2",
```

Comment:
What is `tslib`, what does it do?

Reply:
It's a peer dependency of `@guardian/eslint-config-typescript`.

I've messed around with adding it to [`peerDependencies`](https://nodejs.org/en/blog/npm/peer-dependencies) which just causes more issues, so I've just left it here for now. 

Not sure how to make it more clear, but if you remove it you get warnings from yarn